README.md

```
@@ -5,21 +5,30 @@
 ### Installation
 
 ```sh
-$ yarn
+$ bun
```

markdownlint-cli2 (0.18.1)

Dollar signs used before commands without showing output

(MD014, commands-show-output)